operations/iuf/IUF.md

```diff
@@ -632,6 +641,29 @@ HPE provides management NCN and managed node `sat bootprep` configuration files
 CFS configuration, image, and BOS session template definitions. The administrator may customize the files as needed. The files include
 variables, and the values used are provided by the recipe variables and/or site variables files specified when running `iuf run`.
 
+## Re-executing stages of an IUF session
+
+It is possible to re-execute stages of an IUF session by specifying `iuf run` with the desired stage and other `iuf` arguments. If no changes were made to the product distribution files in the media directory, `iuf run` will
+re-execute any Argo steps that failed during the previous invocation of `iuf run`. Any Argo steps that previously executed successfully will be skipped if possible. If the `-f` argument is specified, all Argo steps will be
+re-executed, regardless of whether they succeeded or failed during the previous invocation of `iuf run`.
+
+### Removing a product from an IUF session
+
+If the administrator wants to remove a product from the IUF session, they must re-execute `iuf run` for the `process-media` stage with the product distribution file and uncompressed content removed from the media directory. This
+removes knowledge of that product from the existing IUF activity.
```

Comment:
"knowledge of that product" feels like it's not the right wording.   Maybe "references to that product" instead?

Reply:
Fixed in next commit.